.github/workflows/danger-pr-labels-local.yml

```diff
@@ -0,0 +1,38 @@
+name: Validated PR Labels (Local Danger)
+on:
+  pull_request:
+    # The restrictions on the types of pull_request event that can trigger this
+    # workflow is intentional. We only want to run label related checks on a
+    # new or reopened PR, or when its labels change.
+    types: [opened, reopened, labeled, unlabeled]
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v1
+        with:
+          node_version: 10.x
+
+      - name: Install Yarn
+        run: npm install -g yarn
+
+      - name: Cache Node Modules
+        id: cache-node-modules
+        uses: actions/cache@v1
+        with:
+          path: node_modules
+          key: ${{ runner.os }}-node_modules-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: ${{ runner.os }}-node_modules
+
+      - name: Yarn Install
+        if: steps.cache-node-modules.outputs.cache-hit != 'true'
+        # frozen-lockfile will make the build fail if the lockfile is not there
+        run: yarn install --frozen-lockfile
+
+      - name: Validate PR Labels
+        run: yarn run danger ci --dangerfile mokagio/danger-settings/org/pr/label.ts
```

Comment:
So this is interesting. I couldn't get the `Dangerfile` from the [peril-settings repo](https://github.com/Automattic/peril-settings/blob/5d22258ca46c3649f2cb863cfac2d16f12c4cba0/org/pr/label.ts) to work, despite it being the one we were using in the previous setup running on the Danger GitHub Action.

I described the problem and my thoughts on it [in this issue in the Danger JS repo](https://github.com/danger/danger-js/issues/992).

I hope there's something I'm missing, but if there isn't, I guess we'll just have to create a new repo that converts all of the `Dangerfile`s in the peril-settings repo to sync ones, like I did in this example, see https://github.com/mokagio/danger-settings/commit/6ffd56f9c60d4d95acff6c6391ac58d95ed45f09.

Reply:
@loremattei @jkmassel I haven't gotten any feedback from the Danger team yet 😕

If you know more TS than I do, maybe you can shed some light on my issue. Or if we can't make the files from peril-settings work, how do you feel about me moving the ownership of the new danger-settings repo to the wordpress-mobile org, so we can keep using remote `Dangerfile`s?

I think using the remote files is a better option than duplicating them across all the repos that might need them. This way, if we make an improvement in a `Dangerfile`, all the repos using it will get it immediately, with no extra work needed on our end.

Reply:
@loremattei @jkmassel update. I sorted it out, all the details on why it wasn't working are [here](https://github.com/danger/danger-js/issues/992#issuecomment-586523281).